code/modules/mob/living/simple_animal/hostile/ordeal/gold.dm

```diff
@@ -273,12 +274,29 @@
 	var/can_act = TRUE
 	var/slash_width = 1
 	var/slash_length = 3
+	var/AOE_cooldown
+	var/AOE_cooldown_time = 10 SECONDS
+	var/AOE_damage = 60
```

Comment:
This is a noon that does a 2 tile pale AOE and it does 60? 
This is a noon; reminder that most HE armor will take 48% from this.
On top of this, you only get 1.04 seconds to react after reaction time

I'd reduce from 60 to 50, this makes you take 40% damage from this, leaving you with 20% of your HP after two hits.
This is a long fight, so I think this is fair.

Reply:
Sure.